app/views/helpers/display-helper.js

```diff
@@ -57,3 +57,11 @@ for (var prisonKey in prisonsEnum) {
 module.exports.getPrisonsByRegion = function (region) {
   return prisonsByRegion[region]
 }
+
+module.exports.toCurrency = function (value) {
+  if (value && value < 0) {
+    return `-£${Number(value * -1).toFixed(2)}`
+  } else {
+    return `£${Number(value).toFixed(2)}`
```

Comment:
If you look at how get claim summary was doing it you need a check to only make it fixed to 2 if there is decimals, otherwise Chris wanted it to say £5 rather than now when it says £5.00